src/IdeHelper/BelongsToThroughRelationsHook.php

```diff
@@ -42,6 +42,9 @@ public function run(ModelsCommand $command, Model $model): void
         }
     }
 
+    /**
+     * @param \Illuminate\Database\Eloquent\Relations\Relation<\Illuminate\Database\Eloquent\Model> $relationship
```

Comment:
You're missing the other two templates:

This should just be:


```
@param \Illuminate\Database\Eloquent\Relations\Relation<*, *, *> $relationship
```

Reply:
What does the `*` mean/do in this case?

Reply:
`*` is a wildcard that means any type is accepted, the template types are already given / scoped on the actual class so there's no need to repeat here. All this is saying is "I need a Relation and I don't care what any of the inner types are".